### PR TITLE
6489 migrate all medic docs references

### PR DESCRIPTION
**Pull request by ronna**
# Description

Replace references to medic-docs and link to the appropriate location in the CHT docs site

medic/cht-core#6489

# Code review items

- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
Hi @ronna 
Thank you for the contribution, this is great! I don't have additional feedback, maybe @abbyad can have a look for the final thumbs up on the PR? 🙂

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -88,9 +88,9 @@ If you believe you've found a security vulnerability in one of our products or p
 
 ### Improving our documentation
 
-**Note:** medic-docs does not involve release management and acceptance testing. Help us maintain the quality of our documentation by submitting a pull request (PR) with any suggested changes. 
+**Note:** medic-docs does not involve release management and acceptance testing. Help us maintain the quality of our documentation by submitting a pull request (PR) with any suggested changes.
 
-Is our documentation up to date? Have we covered everything we should? Could our wording be improved? Read our [Documentation Style Guide](https://github.com/medic/medic-docs/blob/master/development/docs-style-guide.md) then open a pull request with your suggested changes or additions.
+Is our documentation up to date? Have we covered everything we should? Could our wording be improved? Read our [Documentation Style Guide](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/) then open a pull request with your suggested changes or additions.
 Want to talk about Documentation generally? Join our [Community Forum](https://communityhealthtoolkit.org/forum)!
```
Comment (abbyad) [inline CONTRIBUTING.md:94]: While we are here, this section is a duplicate of one above and should be removed. The only difference is the note part, but that is no longer relevant.

**Review by abbyad — APPROVED** [review]
Looks great @ronna!! Ready for merge. We can fix the duplicate section separately.

**Comment by abbyad** [comment]
@ronna the PR is still in Draft mode. Can you confirm that this PR is ready to merge?

**Comment by ronna** [comment]
> @ronna the PR is still in Draft mode. Can you confirm that this PR is ready to merge?
Hi @abbyad yes it's ready to merge. I will create an issue about the duplicate section and work on it

**Comment by abbyad** [comment]
Congrats and thank you @ronna for your first contribution to the `cht-core` repo!!

**Pull request merged**